docs/mutations.md

```diff
@@ -159,6 +159,20 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 | <code>&#124;</code> | `&`                 |
 | `a^b`               | `~(a^b)`            |
 
+## Patterns inside is and switch expressions (_pattern_)
+| Original | Mutated |
+| ------------- | ------------- |
+| `>` | `<` |
+| `>` | `>=` |
+| `>=` | `<` |
+| `>=` | `>` |
+| `<` | `>` |
+| `<` | `<=` |
+| `<=` | `>` |
+| `<=` | `<` |
```

Comment:
I wouldn't say these are new mutations. These are just Equality Operator mutations but in a different context. I'd say we add the `and` and `or` mutations to the Equality Operators chapter. @rouke-broersma agree?

Reply:
We should check with the other stryker teams if they already have similar mutations and in which category they place them

Reply:
@richardwerkman @rouke-broersma Imo, it's better to separate pattern mutations from simple operators, even if they look similar. If you want to implement mutations for other patterns in the future, you will need the separate doc chapter anyway.
But of course it's up to you guys.

Reply:
I've checked with stryker4s as they use a lot of pattern matching. They place these type of mutation in equality operators. So I'd say we keep in line with them.

Reply:
@richardwerkman Ok, thank you! I'll update the PR correspondingly.

Reply:
Docs are updated